src/Console/TinkerCommand.php

```diff
@@ -43,9 +44,8 @@ public function handle()
     {
         $this->getApplication()->setCatchExceptions(false);
 
-        $config = new Configuration([
-            'updateCheck' => 'never',
-        ]);
+        $config = Configuration::fromInput($this->input);
```

Comment:
You may want to use `Configuration::getInputOptions` to ensure that compatible options are bound to `$this->input`.